content/init-alicloud.md

```diff
@@ -0,0 +1,157 @@
+    * **Security group name**: bosh
+    * **Description**: BOSH deployed VMs
+    * **VPC**: Select the "bosh" VPC that you created in [Create a Virtual Private Cloud](#create-vpc).
+
+1. Select the created security group with group name "bosh", in the Actions column click Configure Rules.
+
+1. On the Security Group Rules page, click Add Security Group Rules.
+
+1. Fill out the Edit inbound rules form and click **Save**.
+
+    !!! note
+        It highly discouraged to run any production environment with <code>0.0.0.0/0</code> source or to make any BOSH management ports publicly accessible.
+
+    <table border="1" class="nice">
+      <tr>
+        <th>Type</th>
+        <th>Port Range</th>
+        <th>Source</th>
+        <th>Purpose</th>
+      </tr>
+
+      <tr><td>Custom TCP Rule</td><td>22</td><td>(My IP)</td><td>SSH access from CLI</td></tr>
+      <tr><td>Custom TCP Rule</td><td>6868</td><td>(My IP)</td><td>BOSH Agent access from CLI</td></tr>
+      <tr><td>Custom TCP Rule</td><td>25555</td><td>(My IP)</td><td>BOSH Director access from CLI</td></tr>
+
+      <tr><td>All TCP</td><td>0 - 65535</td><td>ID of this security group</td><td>Management and data access</td></tr>
+      <tr><td>All UDP</td><td>0 - 65535</td><td>ID of this security group</td><td>Management and data access</td></tr>
+    </table>
+
+See [Creating a Security Group](https://www.alibabacloud.com/help/doc-detail/25468.htm)
+
+See [Add security group rules](https://www.alibabacloud.com/help/doc-detail/25471.htm)
+
+---
+## Step 2: Deploy {: #deploy }
+
+1. Install [CLI v2](cli-v2.md).
+
+1. Use `bosh create-env` command to deploy the Director.
+
+    ```shell
+    # Create directory to keep state
+    $ mkdir bosh-1 && cd bosh-1
+
+    # Clone Director templates
+    $ git clone https://github.com/cloudfoundry/bosh-deployment
+
+    # Fill below variables (replace example values) and deploy the Director
+    $ bosh create-env bosh-deployment/bosh.yml \
+        --state=state.json \
+        --vars-store=creds.yml \
+        -o bosh-deployment/alicloud/cpi.yml \
+        -o bosh-deployment/jumpbox-user.yml \
+        -o bosh-deployment/misc/powerdns.yml \
+        -o bosh-deployment/credhub.yml \
+        -o bosh-deployment/uaa.yml \
+        -v dns_recursor_ip=8.8.8.8 \
+        -v director_name=bosh-1 \
+        -v internal_cidr=10.0.0.0/24 \
+        -v internal_gw=10.0.0.1 \
+        -v internal_ip=10.0.0.6 \
+        -v access_key_id=AKI... \
+        -v secret_access_key=wfh28... \
+        -v region=us-east-1 \
+        -v zone=us-east-1a \
+        -v vswitch_id=vsw-rj9rio... \
+        -v security_group_id=sg-rj9dtcbw... \
+        -v key_pair_name=bosh \
+        -v private_key=~/Downloads/bosh.pem
+    ```
+
+    If running above commands outside of an Alicloud VPC, refer to [Exposing environment on a public IP](init-external-ip.md) for additional CLI flags.
+
+    See [Alicloud CPI errors](alicloud-cpi-errors.md) for list of common errors and resolutions.
```

Comment:
The file `alicloud-cpi-errors.md` is missing from this PR.  Please fix, and update the PR.  Thanks.